hydradx/notebooks/Omnipool/LP_fees_analysis.ipynb

```diff
@@ -0,0 +1,269 @@
+      "text/plain": [
+       "<Figure size 1440x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(20, 5))\n",
+    "plt.title('LP fees earned after one year:')\n",
+    "graph_length = len(volume_events[1])\n",
+    "plt.xticks(\n",
+    "    [0, graph_length / 4, graph_length / 2, graph_length * 3/4, graph_length],\n",
```

Comment:
> I don't think it makes sense to have a 0 trade volume run. No trade volume = no fees, I don't think we need to run it or graph it.

True that the zero run is pretty pointless since nothing happens, and we know the result. I'll get rid of the unnecessary simulation and graph line.